docs/tutorials/orc.ipynb

```diff
@@ -0,0 +1,415 @@
+{
```

Comment:
<p>Make this a section, similar to <code>Setup</code>  for ease in navigation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/io/pull/1465/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>